ProcessOut/Classes/ProcessOut.swift

```diff
@@ -497,12 +497,7 @@ public class ProcessOut {
             request.httpMethod = method.rawValue
             request.setValue("application/json", forHTTPHeaderField: "Content-Type")
             request.setValue(authorizationHeader.value, forHTTPHeaderField: authorizationHeader.key)
-            
-            if defaultUserAgent != nil {
-                request.setValue(ProcessOut.defaultUserAgent! + " ProcessOut iOS-Bindings/" + ApiVersion, forHTTPHeaderField: "User-Agent")
-            } else {
-                request.setValue("ProcessOut iOS-Bindings/" + ApiVersion, forHTTPHeaderField: "User-Agent")
-            }
+            request.setValue(ProcessOut.defaultUserAgent + " ProcessOut iOS-Bindings/" + ApiVersion, forHTTPHeaderField: "User-Agent")
```

Comment:
Are you sure this leads to the same behavior as before?

Reply:
We'll lose some information (especially the browser behind the uiwebview) but this is not a main concern as this information could already be wrong (if UIWebView user-agent was different than WKWebview user-agent).

But we'll keep the iOS version and the SDK version.